tests/ui/const-generics/adt_const_params/const_param_ty_bad.stderr

```diff
@@ -11,6 +11,10 @@ note: required by a bound in `check`
    |
 LL | fn check(_: impl std::marker::ConstParamTy) {}
    |                  ^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `check`
+help: use parentheses to call this function
```

Comment:
This is a consequence of us using the trait solving infrastructure. I guess I could suppress it if it's terrible... :thinking:

Reply:
I think this is fine, it's a reasonable suggestion since calling it _does_ make the code work